recipes/torch-geometric/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "torch_geometric" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: py{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 131359fb1539ce5c837e8eb2dce8f7abd75294ae0ce44963648cd8646cfde6e1
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```

It's okay if Windows CI fails for noarch packages with missing dependencies
```suggestion
  noarch: python
```

---

thanks, just updated